### PR TITLE
fix: Use get

### PR DESCRIPTION
**Pull request by Crash--**
🤕

### CODE REVIEW

**Comment by ptbrowne** [comment]
It seems this component could use a test if it can break. BTW I thought it was supposed to be renamed to SupportUs ?

File: src/components/pushClient/Supportus.jsx [hunk 1]
```diff
@@ -24,17 +24,13 @@ const Supportus = ({ t, client }) => {
     'context.data.attributes.enable_premium_links'
   )
   const diskUsage = withDiskUsage(client)
+  const quota = get(diskUsage, 'diskUsage.data.attributes.quota', false)
```
Comment (y-lohse) [inline src/components/pushClient/Supportus.jsx]:
```suggestion
  const quota = get(diskUsage, 'data.attributes.quota', false)
```

I think..?

**Comment by Crash--** [comment]
> It seems this component could use a test if it can break. BTW I thought it was supposed to be renamed to SupportUs ?

I don't consider a NIT as a change to be done.

**Comment by Crash--** [comment]
thanks @y-lohse ! 😓

**Pull request merged**